src/components/suggest/suggest.tsx

```diff
@@ -59,6 +59,9 @@ export const EuiSuggest: FunctionComponent<EuiSuggestProps> = (
       type={item.type}
       key={index}
       label={item.label}
+      labelDisplay={item.labelDisplay}
+      descriptionDisplay={item.descriptionDisplay}
+      labelWidth={item.labelWidth}
       onClick={onItemClick ? () => onItemClick(item) : undefined}
       description={item.description}
```

Comment:
To ensure all item props are passed down to the EuiSuggestItem, we can use the spread method to simplify things. This will then pass down all the props within the item object to the component.

```suggestion
      key={index}
      onClick={onItemClick ? () => onItemClick(item) : undefined}
      {...item}
```

Reply:
It slipped my mind to do this. When I made the change I was getting a TS error but our TS maven Greg helped me out.